src/core/thread/dua_manager.cpp

```diff
@@ -519,6 +524,7 @@ void DuaManager::PerformNextRegistration(void)
 
     mIsDuaPending   = true;
     mRegisteringDua = dua;
+    mDelay.mValue   = 0;
```

Comment:
Should we call UpdateTimeTickerRegistration() immediately?

Reply:
ditto